package.json

```diff
@@ -54,7 +54,7 @@
     "styled-components": "^5.2.3",
     "swr": "^1.1.1",
     "utf-8-validate": "^5.0.7",
-    "wagmi": "^0.5.12",
+    "wagmi": "^0.6.1",
```

Comment:
main change that fixed initial build issue below; this revealed other issues with sentry:

![image](https://user-images.githubusercontent.com/12162433/183233497-e9a4c0ca-f365-4c9c-ba72-d45d4e53ab16.png)